modules/ROOT/pages/troubleshoot-guide.adoc

```diff
@@ -45,6 +45,75 @@ tcp://dias-ingestor-nginx.prod.cloudhub.io:5044 ✔
  
 If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have completed the required whitelisting as described in xref:install-prereqs#port-ip-addresses-and-hostnames-to-whitelist[Port IPs and Hostnames to Whitelist].
 
+
+== Troubleshoot Chrony Synchronization Check Failure
+
+Runtime Fabric requires chrony for its time synchronization daemon. If chrony is not found by the `init.sh` installation script, Runtime Fabric installation fails with the following error:
+```
+============================================================================================
+1 / 10: Install required packages
+================================================
+chrony-3.4-1.el7.x86_64
+Checking chrony sync status...Retrying in 30 seconds...
+Retrying in 30 seconds...
+Error: chrony sync check failed 3 times, giving up.
+
+***********************************************************
+** Oh no! Your installation has stopped due to an error. **
+***********************************************************
+1. Visit the troubleshooting guide for help:
+https://docs.mulesoft.com/runtime-fabric/latest/troubleshoot-guide#install-required-packages
+
+2. Resume installation by running /opt/anypoint/runtimefabric/init.sh
+
+Additional information: Error code: 1; Step: install_required_packages; Line: -;
+```
+
+Perform the following steps to fix this issue:
+
+. Verify that chrony is enabled:
++
+`systemctl enable chronyd`
+. Verify that the  Network Time Protocol (NTP) is disabled and is not running:
++
+`systemctl stop ntpd; systemctl disable ntpd`
+. Contact your network team to verify that the time servers in `/etc/chrony.conf` are reachable.
```

Comment:
Will all audiences understand who is meant by "your network team"?
Is this inside MuleSoft, or outside, or both?